Exception thrown when opening idea

Version: 4

Happens everytime I open Idea
```
Unhandled exception in [no parent and no name, StandaloneCoroutine{Cancelling}@a450226, Dispatchers.Default]

java.lang.RuntimeException: Cannot invoke (class=ProjectToolWindowGearActionDecorator, method=stateChanged, topic=tool window events)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:639)
	at com.intellij.util.messages.impl.MessageBusImplKt.deliverMessage(MessageBusImpl.kt:395)
	at com.intellij.util.messages.impl.MessageBusImplKt.pumpWaiting(MessageBusImpl.kt:374)
	at com.intellij.util.messages.impl.MessageBusImplKt.access$pumpWaiting(MessageBusImpl.kt:1)
	at com.intellij.util.messages.impl.MessagePublisher.invoke(MessageBusImpl.kt:433)
	at jdk.proxy1/jdk.proxy1.$Proxy109.stateChanged(Unknown Source)
	at com.intellij.openapi.wm.impl.ToolWindowManagerImpl.fireStateChanged(ToolWindowManagerImpl.kt:1689)
	at com.intellij.openapi.wm.impl.ToolWindowManagerImpl.registerToolWindow(ToolWindowManagerImpl.kt:1026)
	at com.intellij.dependencytoolwindow.DependencyToolWindowFactory$execute$2.invokeSuspend(DependencyToolWindowFactory.kt:54)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:209)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:191)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:838)
	at com.intellij.openapi.application.impl.ApplicationImpl$3.run(ApplicationImpl.java:454)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:74)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:114)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:779)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:749)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:909)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:756)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$5(IdeEventQueue.java:437)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:436)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:615)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:434)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:838)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:480)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [no parent and no name, StandaloneCoroutine{Cancelled}@a450226, Dispatchers.Default]
Caused by: java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "com.intellij.ui.content.ContentManager.getContentCount()" because "this.myContentManager" is null
	at com.github.novotnyr.mavenversion.ProjectToolWindowGearActionDecorator.getDefaultProjectViewActions(ProjectToolWindowGearActionDecorator.java:57)
	at java.base/java.util.Optional.map(Optional.java:260)
	at com.github.novotnyr.mavenversion.ProjectToolWindowGearActionDecorator.decorateToolWindow(ProjectToolWindowGearActionDecorator.java:37)
	at com.github.novotnyr.mavenversion.ProjectToolWindowGearActionDecorator.stateChanged(ProjectToolWindowGearActionDecorator.java:31)
	at com.intellij.openapi.wm.ex.ToolWindowManagerListener.stateChanged(ToolWindowManagerListener.java:41)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeMethod(MessageBusImpl.kt:655)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:621)
	... 41 more
Caused by: java.lang.NullPointerException: Cannot invoke "com.intellij.ui.content.ContentManager.getContentCount()" because "this.myContentManager" is null
	at com.intellij.ide.projectView.impl.ProjectViewImpl.getActions(ProjectViewImpl.java:620)
	at com.github.novotnyr.mavenversion.ProjectToolWindowGearActionDecorator.getDefaultProjectViewActions(ProjectToolWindowGearActionDecorator.java:54)
	... 47 more
```

--- a/src/main/java/com/github/novotnyr/mavenversion/ProjectToolWindowGearActionDecorator.java
+++ b/src/main/java/com/github/novotnyr/mavenversion/ProjectToolWindowGearActionDecorator.java
@@ -54,7 +54,8 @@ public class ProjectToolWindowGearActionDecorator implements ToolWindowManagerLi
                 var providerActions = actionProvider.getActions(false);
                 defaultActions.addAll(providerActions);
             } catch (Exception e) {
-                throw new RuntimeException(e);
+                // Some versions of platform trigger: Cannot invoke "com.intellij.ui.content.ContentManager.getContentCount()" because "this.myContentManager" is null
+                // Let's ignore this error and return empty actions
             }
         }
         return defaultActions;

